site/pt/_partials/devtools/whats-new.md

```diff
@@ -6,26 +6,26 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
 
 {# ### Chrome 111 {: #chrome111 }
 
-* [Debugging HD color with the Styles pane](/pt/blog/new-in-devtools-111/#color)
-* [Enhanced breakpoint UX](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
-* [Customizable Recorder shortcuts](/pt/blog/new-in-devtools-111/#recorder)
-* [Better syntax highlight for Angular](/pt/blog/new-in-devtools-111/#syntax)
-* [Reorganize caches in the Application panel](/pt/blog/new-in-devtools-111/#cache)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-111/#misc)
+* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
+* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
+* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)
+* [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)
+* [Paineis de caches reorganizador na aba Aplicação](/pt/blog/new-in-devtools-111/#cache)
+* [Outras novidades](/pt/blog/new-in-devtools-111/#misc)
  #}
 
 
 {# ### Chrome 110 {: #chrome110 }
 
-* [Clearing Performance Panel on reload](/pt/blog/new-in-devtools-110/#perf)
-* [Recorder updates](/pt/blog/new-in-devtools-110/#recorder)
-* [View and highlight the code of your user flow in the Recorder](/pt/blog/new-in-devtools-110/#recorder-code)
-* [Customize selector types of a recording](/pt/blog/new-in-devtools-110/#recorder-selector)
-* [Edit user flow while recording](/pt/blog/new-in-devtools-110/#recorder-edit)
-* [Automatic in-place pretty print](/pt/blog/new-in-devtools-110/#pretty-print)
-* [Better syntax highlight and inline preview for Vue, SCSS and more](/pt/blog/new-in-devtools-110/#highlight)
-* [Ergonomic and consistent Autocomplete in the Console](/pt/blog/new-in-devtools-110/#console)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-110/#misc)
+* [Limpeza do painel Performance ao recarregar](/pt/blog/new-in-devtools-110/#perf)
+* [Novidades no Gravador](/pt/blog/new-in-devtools-110/#recorder)
+* [Veja e realçe códigos do fluxo do usuário no Gravador](/pt/blog/new-in-devtools-110/#recorder-code)
+* [Customize os tipos de seletores de uma gravação](/pt/blog/new-in-devtools-110/#recorder-selector)
+* [Edição de fluxo de usuário enquanto grava](/pt/blog/new-in-devtools-110/#recorder-edit)
+* [Auto formatação de código na mesma aba](/pt/blog/new-in-devtools-110/#pretty-print)
+* [Melhor syntax highlight para Vue, SCSS e mais](/pt/blog/new-in-devtools-110/#highlight)
+* [Autocomplete consistente no console](/pt/blog/new-in-devtools-110/#console)
```

Comment:
"Autocomplete ergonômico e consistente no Console" - adding ergonomic, which was dropped and Console in uppercase (since it refers to the feature, not a console)